### PR TITLE
standardize on non-US-influenced parameter names

### PR DESCRIPTION
**Pull request by trescube**
switched from `city` and `state` to `locality` and `region`, respectively

### CODE REVIEW

File: sanitizer/_synthesize_analysis.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 const _ = require('lodash');
 
-const fields = ['address', 'neighbourhood', 'borough', 'city', 'county',
-  'state', 'postalcode', 'country'];
+const fields = {
+  'address': 'address',
+  'neighbourhood': 'neighbourhood',
+  'borough': 'borough',
+  'locality': 'city',
```
Comment (orangejulius) [inline sanitizer/_synthesize_analysis.js]: Wait, why do we need this? What part still uses city?

Comment (trescube) [inline sanitizer/_synthesize_analysis.js]: We adopted the libpostal names for our internal representation:

```
> lancaster, pa

Result:

{
  "city": "lancaster",
  "state": "pa"
}
```

This change is just for outward-facing behavior.  People shouldn't be coding against our internal analysis representation but that should be changed at some point, though it's a bigger effort than what's been done here.

Comment (trescube) [inline sanitizer/_synthesize_analysis.js]: Here's our internal representation:

```
"query": {
"text": "lancaster, pa",
"parsed_text": {
"city": "lancaster",
"state": "pa"
},
"size": 10,
"private": false,
"querySize": 20
}
```

Comment (orangejulius) [inline sanitizer/_synthesize_analysis.js]: oh, right. that makes sense.

**Pull request merged**